### PR TITLE
[WIP]Translate into Chinese

### PR DESCRIPTION
**Pull request by sjfkai**
Translated all tips.

### CODE REVIEW

**Comment by loverajoel** [comment]
@sjfkai awesome! 

I need your help and you have to follow [this](https://github.com/loverajoel/jstips/pull/173) "instructions" to adapt the defaults vars of the site to Chinese, after that, we can merge this!

Example https://github.com/loverajoel/jstips/pull/178

**Comment by icescut** [comment]
where can I see the Chinese version?

**Comment by sjfkai** [comment]
@icescut  It have not merged Chinese version into the offical blog. You can temporarily look at [this](https://cnodejs.org/topic/56a050ac8392272262331d62#56aa02f971204e03637a36a0).

File: _config.yml [hunk 1]
```diff
@@ -125,3 +136,20 @@ transl:
     footer_placeholder_subscribe: your@email.com
     arrow_prev: Previous Tip
     arrow_next: Next Tip
+  zh_CN:
```
Comment (sjfkai) [inline _config.yml:139]: @loverajoel  I think I have adapt the defaults vars here. If it was anything wrong. Please tell me. :smile:

**Comment by icescut** [comment]
@sjfkai thx :+1:

File: _config.yml (hunk 1 above)
Comment (loverajoel) [inline _config.yml:139]: perfect!

**Comment by loverajoel** [comment]
@sjfkai what is the translation of 'Chinese" word in Chinese?

**Comment by ghost** [comment]
`中文` is the translation of Chinese in Chinese.

**Comment by loverajoel** [comment]
@li-xinyang thanks!

**Comment by loverajoel** [comment]
Guys, I merged this branch to master! Congrats for huge work!

http://www.jstips.co/zh_CN/

@sjfkai I added you how collaborator of the repository and now you are in charge of the Chinese version.

Hope this help to grow the community!
Please let me know any question.

Thanks for your efforts!!

**Comment by loverajoel** [comment]
Also I crate a new label to tag pr and issues related to this language

**Pull request merged**